pages/import_issues_page_col_set.php

```diff
@@ -125,7 +125,7 @@
 			echo sprintf( plugin_lang_get( 'column_number' ), $i + 1);
 		}
 		else {
-			echo prepare_output($t_column_title[$i]);
+			echo $t_column_title[$i];
```

Comment:
@vboctor this output is already convert as UTF-8,if use prepare_output() again will make the previous processing in vain